test/onnx/test_fx_op_consistency.py

```diff
@@ -79,6 +181,214 @@
 # fmt: on
 
 SKIP_XFAIL_SUBTESTS: tuple[onnx_test_common.DecorateMeta, ...] = (
+    xfail(
+        "acos",
+        matcher=lambda sample: sample.input.dtype == torch.float64,
```

Comment:
Do we need this or it the case already covered by what’s above?

Reply:
All dtype related should be in the first list?

Reply:
I just didn't see a need to match subtests if it will effectively skip the whole test? FYI it's possible to have multiple xfail/skip entries for the same opinfo name.

Reply:
Sure

Reply:
Done